src/events/cd-user-ticket-list-item.vue

```diff
@@ -0,0 +1,295 @@
+          });
+        });
+        return totalEventCapacity === totalTicketsBooked;
+      },
+      nextStartTime() {
+        return EventsUtil.getNextStartTime(this.event);
+      },
+      isRecurring() {
+        return EventsUtil.isRecurring(this.event);
+      },
+      recurringFrequencyInfo() {
+        return EventsUtil.buildRecurringFrequencyInfo(this.event);
+      },
+      isPastEvent() {
+        const now = moment();
+        const eventStartTime = moment(this.event.dates[this.event.dates.length - 1].startTime);
+        eventStartTime.subtract(eventStartTime.utcOffset(), 'minutes');
+        return now.isAfter(eventStartTime);
+      },
+      bookLink() {
+        return `/dojo/${this.dojo.id}/event/${this.event.id}`;
+      },
+      formattedStartTime() {
+        return this.$options.filters.cdTimeFormatter(this.event.dates[0].startTime);
+      },
+      formattedEndTime() {
+        return this.$options.filters.cdTimeFormatter(this.event.dates[0].endTime);
+      },
+      formattedFirstDate() {
+        const sortedDates = sortBy(this.event.dates, date => date.startTime);
+        return this.$options.filters.cdDateFormatter(sortedDates[0].startTime);
+      },
+      formattedLastDate() {
+        const sortedDates = sortBy(this.event.dates, date => date.startTime);
+        return this.$options.filters.cdDateFormatter(sortedDates[sortedDates.length - 1].startTime);
+      },
+    },
+    methods: {
+      prepareSessions() {
+        this.event.sessions.forEach((s) => {
+          this.sessions[s.id] = s;
+        });
+      },
+      prepareTickets() {
+        this.event.sessions.forEach((s) => {
+          s.tickets.forEach((t) => {
+            this.tickets[t.id] = t;
+          }, this);
+        });
+      },
+      async cancel() {
+        const payload = [];
+        const commonFields = ['dojoId', 'eventId', 'sessionId', 'ticketName',
+          'ticketType', 'ticketId', 'userId', 'notes', 'deleted'];
+        this.applications.forEach((application) => {
+          application.deleted = true; // eslint-disable-line no-param-reassign
+          payload.push(pick(application, commonFields));
+        });
+        await EventService.manageTickets(payload);
+        this.applications = [];
+        alert(this.$t('Applications cancelled')); // eslint-disable-line no-alert
+      },
+      async loadUsersApplications() {
+        const res = await EventService.loadApplications(this.event.id);
+        if (res.body && res.body.length > 0) {
+          this.applications = res.body;
+        }
+      },
+    },
+    components: {
+      cdAttendee,
+    },
+    async created() {
+      this.prepareSessions();
+      this.prepareTickets();
+      this.loadUsersApplications();
+    },
+    filters: {
+      cdDateFormatter,
+      cdTimeFormatter,
+      cdUrlFormatter,
+    },
+  };
+</script>
+<style scoped lang="less">
+    @import "../common/variables";
```

Comment:
Let's nitpick! Two too many spaces! :trollface:

Reply:
How come it hasn't been caught by the linter?